client/state/sites/selectors.js

```diff
@@ -63,18 +62,40 @@ export const getSite = createSelector(
 
 		return {
 			...site,
-			...getComputedAttributes( site ),
 			...getJetpackComputedAttributes( state, siteId ),
 			hasConflict: isSiteConflicting( state, siteId ),
 			title: getSiteTitle( state, siteId ),
 			slug: getSiteSlug( state, siteId ),
 			domain: getSiteDomain( state, siteId ),
-			is_previewable: isSitePreviewable( state, siteId )
+			is_previewable: isSitePreviewable( state, siteId ),
+			options: getSiteOptions( state, siteId ),
 		};
 	},
 	( state ) => state.sites.items
 );
 
+export function getSiteOptions( state, siteId ) {
+	const site = getRawSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+
+	const isWpcomMappedDomain = getSiteOption( state, siteId, 'is_mapped_domain' ) && ! isJetpackSite( state, siteId );
+	const wpcomUrl = withoutHttp( getSiteOption( state, siteId, 'unmapped_url' ) );
+
+	// The 'standard' post format is saved as an option of '0'
+	let defaultPostFormat = getSiteOption( state, siteId, 'default_post_format' );
+	if ( ! defaultPostFormat || defaultPostFormat === '0' ) {
+		defaultPostFormat = 'standard';
+	}
+
+	return {
+		...site.options,
+		...isWpcomMappedDomain && { wpcom_url: wpcomUrl },
+		default_post_format: defaultPostFormat
+	};
+}
+
```

Comment:
First, I don't feel strongly on this so if you really think this is a blocker I can simply move it to `state/selectors`.

My logic is that all the selectors used to assemble the site in the `getSite` selector currently lives in that file and I'm not sure we plan to even use this selector at all or if we will always just access `options` through the property of the `site`.
Whereas we keep it as a computed property or not is probably a subject for another PR since I haven't started to update usage of site.options anywhere (and I'm not sure we should).

In the case of `is_customizable`, I was able to update all usage to use the new selector instead and removed it from the site computed properties, so it's definitely a valid/useful selector and this is why it was moved to `state/selectors`.